components/brave_ads/core/internal/ad_switches.cc

```diff
@@ -0,0 +1,12 @@
+/* Copyright (c) 2023 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#include "brave/components/brave_ads/core/ad_switches.h"  // IWYU pragma: keep
+
+namespace brave_ads::switches {
+
+const char kAdAffectFeatures[] = "ad-affect-features";
```

Comment:
nit: maybe `kFeatureSwitch` due to namespacing of `brave_ads::switches::kFeatureSwitch` and `brave-ad-features` instead of `ad-affect-features` for the switch name?